templates/ingress-gateways-deployment.yaml

```diff
@@ -368,11 +370,11 @@ spec:
                   - "/bin/sh"
                   - "-ec"
                   - |
-                      /consul-bin/consul services deregister \
-                      {{- if $root.Values.global.enableConsulNamespaces }}
-                      -namespace={{ default $defaults.consulNamespace .consulNamespace }} \
-                      {{- end }}
-                      -id="${POD_NAME}"
+                    /consul-bin/consul services deregister \
+                    {{- if $root.Values.global.enableConsulNamespaces }}
+                    -namespace={{ default $defaults.consulNamespace .consulNamespace }} \
+                    {{- end }}
+                    -id="${POD_NAME}"
```

Comment:
should be indented to match style elsewhere. I think we could do with a full YAML format maybe to standardize but should be a sep PR